openbb_terminal/DOCKER_README.md

```diff
@@ -151,7 +151,7 @@ xhost + $IP
 Now we can run the docker container, adding the display to the environment:
 
 ```bach
```

Comment:
<img width="276" alt="image" src="https://user-images.githubusercontent.com/11668535/198881794-33382117-3be6-4941-a66c-69f3cd636086.png">

Reply:
@RCasatta can you please change bach to bash? Clearly there was a typo there and since we're looking it it - let's go to the end =)

I think we should be merging this one after bach is gone